_posts/2022-04-19-small-development-grant.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title:  "NumFOCUS small development grant: adding complex number support to JuMP"
+date:   2022-04-19
+categories: [announcements]
+---
+
+The [JuMP Steering Committee](/pages/governance/#steering-committee) is pleased
+to announce that we have received a
+[small development grant](https://numfocus.org/programs/small-development-grants)
+from [NumFOCUS](https://numfocus.org) to add complex number support to JuMP.
+
+Currently, JuMP and MathOptInterface are limited to formulating optimization
+problems with real-valued decision variables. However, complex numbers appear in
+a variety of industrial optimization problems. One of the most important of these is
+the AC optimal power flow problem (AC-OPF), which is used world-wide to optimize
+the operations of electrical power grids. Currently, users in JuMP are forced to
+decompose their problem into the real and complex parts. This leads to a
+duplication in which twice as many variables are needed, reducing readability
+and increasing the likelihood of bugs.
+
+Complex number support has been a common feature request for users of JuMP, and
+many users leave JuMP for alternative tools which have complex-number support. In
```

Comment:
"many users leave JuMP"
What is meant by "many"? 10s? 100s? 1000s?

It doesn't seem necessary to put emphasis on a negative here. It's implied that users requesting this feature might decide to use other tools that provide it.